src/adaptations/device-layer/include/Weave/DeviceLayer/EFR32/freertos_bluetooth.h

```diff
@@ -0,0 +1,91 @@
+/***************************************************************************/ /**
```

Comment:
This file needs an OpenWeave licence header.

Reply:
Done